P2-Project_Outline/README.md

```diff
@@ -1,17 +1,45 @@
 # Project Outline
 For this assignment, you will submit a high-level outline of your project. This can, and likely will, change over time. In particular, your mentor will provide feedback and direction and feedback to help sharpen your ideas. So don't worry if you feel unsure about some aspects of the outline, or if you have to change some things later.
 
+
 ## Assignment Description
 [Project Outline Assignment](https://education.launchcode.org/liftoff/assignments/project-outline/)
 
 ## Submission Instructions
 
 ### Overview
 Include overview here
+
+My idea for the capstone project is a website that will help me keep track of my dog sitting clients. It will allow
+users to signup and login as well as create a profile. Users will be able to add information such as: phone number,
+address, email, number of pets and their names, dietary restrictions, health history for pets, the vet information
+for the pet. I am also wanting to create a way that clients can message me and request dates which they would like
+me to dogsit.
+
 ### Features
 Include Features here
+
+User login: Users will be able to create an account and login. Each user will have a profile page with their information
+and the information of their pets
+
+Reviews: Users will be able to leave reviews so that other users can view and get a sense of the customer satisfaction.
+
+Messaging: Users will be able to message me with requests for dates to dogsit.
+
 ### Technologies
 Include Technologies here
 
+Java
+Thymeleaf
+SpringBoot
+MySQL
+
 ### What I'll Have to Learn
 Include what you will need to learn here
+<<<<<<< HEAD
```

Comment:
To resolve this merge conflict, just delete lines 39, 44, and 45! Then re-commit and push up your changes. (No need to create a new pull request; your changes will appear so long as you commit to the same branch.)